go.mod

```diff
@@ -9,15 +9,15 @@ require (
 	github.com/gosuri/uilive v0.0.4
 	github.com/joho/godotenv v1.4.0
 	github.com/manifoldco/promptui v0.9.0
-	github.com/onflow/cadence v0.23.3
-	github.com/onflow/cadence/languageserver v0.18.3-0.20220202133308-207188a51831
+	github.com/onflow/cadence v0.21.3-0.20220601002855-8b113c539a2c
```

Comment:
just curious here, why are cadence beeing downgraded?

Reply:
It's not, it's just how go.mod keeps a track of versions when you target a specific commit, but the commit that's targeted here is actually this one https://github.com/onflow/cadence/commit/8b113c539a2cfbbb0c6311c04eb3986f3db418b6 which is latest secure cadence tag, we could use the tag instead tho, but we were doing it like this for the secure cadence updates. Will anyhow make another update as soon as non beta tag is done.

Reply:
Awesome! I tested this version in overflow and it works great! Vim langauge-server even gives me the correct help suggetions.